kalite/distributed/templates/distributed/base_manage.html

```diff
@@ -14,13 +14,13 @@
         <ul class="nav nav-tabs">
             <div class="mobile-manage-nav">
             {% if request.session.facility_user.facility.id %}
-                <li class="teacher-only users {% block user_active %}{% endblock user_active %}"><a href="{% url 'facility_management' zone_id=None facility_id=request.session.facility_user.facility.id %}" title="{% trans 'Manage Users' %}"><div><span aria-hidden="true"><i class="icon icon-uniE606"></i></span><br/>{% trans "Users" %}</div></a></li>
+                <li class="teacher-only users {% block user_active %}{% endblock user_active %}"><a href="{% url 'facility_management' zone_id=None facility_id=request.session.facility_user.facility.id %}" title="{% trans 'Manage Users' %}"><div></i><br/>{% trans "Users" %}</div></a></li>
             {% endif %}
-                <li class="facility {% block facility_active %}{% endblock facility_active %}"><a href="{% url 'zone_management' zone_id=None %}" title="{% trans 'Manage Facilities' %}"><div><span aria-hidden="true"><i class="icon icon-uniE603"></i></span><br/>{% trans "Facilities" %}</div></a></li>
+                <li class="facility {% block facility_active %}{% endblock facility_active %}"><a href="{% url 'zone_management' zone_id=None %}" title="{% trans 'Manage Facilities' %}"><div><i class="icon icon-uniE603"></i><br/>{% trans "Facilities" %}</div></a></li>
             </div>
             <div class="mobile-manage-nav">
-                <li class="video {% block video_active %}{% endblock video_active %}"><a href="{% url 'update_videos' %}" title="{% trans 'Update this server with new videos and languages' %}"><div><span aria-hidden="true"><i class="icon icon-uniE610"></i></span><br/>{% trans "Videos" %}</div></a></li>
```

Comment:
@66eli77 We need those aria-hidden="true" attributes, somewhere. You can't just delete them!